packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -5,27 +5,32 @@ const { toBeAccessible } = require('../commands/expect');
 const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
 
 const hideInputCaret = require('../commands/hide-input-caret');
+const { element } = require('../commands/validates');
 
 class TerraService {
   constructor(options = {}) {
-    const { formFactor, theme } = options;
-
-    this.formFactor = formFactor;
-    this.theme = theme || 'terra-default-theme';
+    const theme = { theme: 'terra-default-theme' };
+    this.serviceOptions = { ...theme, ...options };
```

Comment:
Theme can probably be inlined here rather than created into an object and then deconstructed, but I think these service options might go away

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/464/commits/04718224294139afe8b92e05a16b2910ba550b35